website/_redirects

```diff
@@ -11,16 +11,8 @@
 /segmentation                                        /use-cases/multi-platform-service-mesh            301!
 /segmentation.html                                   /use-cases/multi-platform-service-mesh            301!
 /api.html                                            /api-docs                                         301!
-/docs/agent/acl-rules.html                           /docs/acl/acl-rules                               301!
-/docs/agent/acl-rules                                /docs/acl/acl-rules                               301!
-/docs/agent/acl-system.html                          /docs/acl/acl-system                              301!
-/docs/agent/acl-system                               /docs/acl/acl-system                              301!
```

Comment:
I think we want to keep these origins, but update the destinations to be direct redirects? Like
```
/docs/agent/acl-rules.html                           /docs/security/acl/acl-rules                               301!
/docs/agent/acl-rules                                /docs/security/acl/acl-rules                               301!
/docs/agent/acl-system.html                          /docs/security/acl/acl-system                              301!
/docs/agent/acl-system                               /docs/security/acl/acl-system                              301!
```

Reply:
I added the direct redirects on lines 39-46